rfcs/rfc-business-ledger-data-structure.md

```diff
@@ -0,0 +1,83 @@
+- Start Date: 2021-04-22
+- Target Major Version: [information on a ledger](https://github.com/bcgov/entity/issues/6969)
+- Reference Issues: n/a
+- Entity Issue: github/entity#7193
+- Implementation PR: (leave this empty)
+
+# Summary
+
+Bulk listings of filings should return a catalog of filings, with enough information to be useful, to allow the user to further select the filing of interest.
+
+The filing endpoint should not default to return the complete filing, of every filing on the businesses ledger.
+
+# Basic example
+
+The RESTful style call to ```GET /businesses/{business_identifier}/filings``` will return a cata log of filings.
+
+The RESTful style call to ```GET /businesses/{business_identifier}/filings/{filing_id}``` will return the full individual filing.
+
+The catalog call follows the following pattern:
+``` json
+{"filings":[
+    {"name": "annualReport",
+     "status": "Completed",
+     "effectiveDate": "2003-07-01T00:00:00+00:00",
+     "availableOnPaperOnly": True,
+     ...
+     "filingLink": "https://api.bcregistry.ca/v1/businesses/CP00001/filings/1243",
```

Comment:
This is cool but not required, as the UI can build this URL.
Do you have something future in mind that would benefit from this?

Reply:
it's part of the RESTful approach if you pick up more HATEOS features.
pay and auth have adopted it, so adding to continue to drive consistency.